tests/test_packageAlerts.py

```diff
@@ -265,6 +265,9 @@ def testCreateCcdDataCutout(self):
                                      self.cutoutWcs.wcs.cd)
         self.assertFloatsAlmostEqual(ccdData.data,
                                      calibExposure.getImage().array)
+        self.assertFloatsAlmostEqual(ccdData.psf,
+                                     self.exposure.psf.computeKernelImage(self.center).array)
+        self.assertFloatsAlmostEqual(np.sum(ccdData.psf), 1.0)
```

Comment:
Not sure this last one is necessary: you don't need to check that `computeKernelImage` is normalized here, since it's already well tested within afw.